downstream/modules/platform/ref-accessing-control-auto-hub-eda-control.adoc

```diff
@@ -38,9 +38,13 @@ The automation hub UI is available by default at:
 https://<hub node>:444
 ----
 
+Login as the admin user with the password you created for *controller_admin_password*.
+
 .Accessing Automation EDA UI
```

Comment:
EDA is not an approved acronym. You should replace this with Accessing Event-Driven Ansible UI.